src/lib/isLicensePlate.js

```diff
@@ -12,6 +12,8 @@ const validators = {
     /^[A-Z]{2}[- ]?((\d{3}[- ]?(([A-Z]{2})|T))|(R[- ]?\d{3}))$/.test(str),
   'pt-BR': str =>
     /^[A-Z]{3}[ -]?[0-9][A-Z][0-9]{2}|[A-Z]{3}[ -]?[0-9]{4}$/.test(str),
+  'sv-SE': str =>
+    /(^[ABCDEFGHJKLMNOPRSTUWXYZ]{3}[ ]?[\d]{2}[ABCDEFGHJKLMNOPRSTUWXYZ\d]$)|(^[A-ZÅÄÖ ]{2,7}$)/.test(str),

```

Comment:
I suggest you make your regex more simple without capturing groups and with ranges instead of all possible letters in order to make it easy to read and a little bit more efficient.
Something like this  should be easier to understand: `/^[A-HJ-PR-UW-Z]{3} ?[\d]{2}[A-HJ-PR-UW-Z\d]$|^[A-ZÅÄÖ ]{2,7}$/`

Reply:
Fixed in d84dbe81b5f189274ec601f58c193f908e203322